packages/app/src/cli/models/extensions/specifications/tax_calculation.ts

```diff
@@ -6,6 +6,12 @@ const TaxCalculationsSchema = BaseSchema.extend({
   production_api_base_url: zod.string(),
   benchmark_api_base_url: zod.string().optional(),
   calculate_taxes_api_endpoint: zod.string(),
+  input: zod.object({
```

Comment:
Should we consider naming this more specific such as `metafield_reference`?

Reply:
Changed this to be `input.metafield_identifiers` to match the [Storefront pattern](https://shopify.dev/docs/api/storefront/unstable/input-objects/HasMetafieldsIdentifier).